tests/api/reports-customers.php

```diff
@@ -431,4 +431,43 @@ public function test_customer_retrieval_from_order_bad_order() {
 		$this->assertFalse( CustomersDataStore::get_existing_customer_id_from_order( false ) );
 		$this->assertFalse( CustomersDataStore::get_or_create_customer_from_order( false ) );
 	}
+
+	/**
+	 * Test user deletion.
+	 */
+	public function test_user_deletion() {
+		wp_set_current_user( $this->user );
+
+		// Creating a customer should show up regardless of orders.
+		$customer = WC_Helper_Customer::create_customer( 'deleteme', 'password', 'deleteme@example.com' );
+
+		WC_Helper_Queue::run_all_pending();
+
+		$request = new WP_REST_Request( 'GET', $this->endpoint );
+		$request->set_query_params(
+			array(
+				'per_page' => 1,
+			)
+		);
+		$response  = $this->server->dispatch( $request );
+		$customers = $response->get_data();
+
+		$this->assertEquals( 200, $response->get_status() );
+		$this->assertCount( 1, $customers );
+		$this->assertEquals( $customer->get_id(), $customers[0]['user_id'] );
+
+		// Delete the user associated with the customer.
```

Comment:
Where is the user getting deleted? It obviously is because the test is passing, so wondering what I'm missing.

Reply:
Woah this is some voodoo.. I accidentally removed the line calling `wp_delete_user()` in https://github.com/woocommerce/woocommerce-admin/pull/4977/commits/d715efe283fea4c1f183677411f3c5206fdde084#diff-6b0548e1900dcb4006b4abbd3929eb97L460...

Reply:
Fixed that up in 43325e4.